pkg/webhook/mutation/pod/v2/attributes.go

```diff
@@ -0,0 +1,101 @@
+package v2
+
+import (
+	"fmt"
+	"regexp"
+
+	containerattr "github.com/Dynatrace/dynatrace-bootstrapper/pkg/configure/attributes/container"
+	podattr "github.com/Dynatrace/dynatrace-bootstrapper/pkg/configure/attributes/pod"
+	"github.com/Dynatrace/dynatrace-operator/pkg/consts"
+	"github.com/Dynatrace/dynatrace-operator/pkg/util/kubeobjects/env"
+	"github.com/Dynatrace/dynatrace-operator/pkg/util/kubeobjects/mounts"
+	dtwebhook "github.com/Dynatrace/dynatrace-operator/pkg/webhook"
+	"github.com/Dynatrace/dynatrace-operator/pkg/webhook/mutation/pod/v2/common/volumes"
+	"github.com/Dynatrace/dynatrace-operator/pkg/webhook/mutation/pod/v2/metadata"
+	corev1 "k8s.io/api/core/v1"
+)
+
+func (wh *Injector) addPodAttributes(request *dtwebhook.MutationRequest) error {
+	attr := podattr.Attributes{
+		PodInfo: podattr.PodInfo{
+			PodName:       createEnvVarRef(consts.K8sPodNameEnv),
+			PodUid:        createEnvVarRef(consts.K8sPodUIDEnv),
+			NamespaceName: request.Pod.Namespace,
+		},
+		ClusterInfo: podattr.ClusterInfo{
+			ClusterUId:      request.DynaKube.Status.KubeSystemUUID,
```

Comment:
```suggestion
			ClusterUId:      request.DynaKube.Status.KubeSystemUUID,
```
different style of naming, `PodUid` and `ClusterUId` ?if possible to rename later to `ClusterUID` and `ClusterUID`

Reply:
different project, so not now

Reply:
yeah i know, that's why just don't forget about it :)